### ISSUE TITLE
Locating Synopsis Data

### ISSUE THREAD

**Issue by varungupta31** [movienet/movienet-tools#40]
I downloaded the `MovieNet.tar.gz` file containing the following zip files
`annotation.v1.zip  meta.v1.zip  script1K.v1.zip  subtitle1K.v1.zip`
and some other JSON files, but none of these contain the synopsis information.

Is synopsis part of the larger download file only? Is there a way to only get the synopsis data?

**Comment by ycxioooong** [comment]
Hi there, synopsis information is contained in annotation.v1.zip

**Comment by varungupta31** [comment]
Hi, thanks for the response.
I checked `annotation.v1.zip` again, it has `8918` `.json` files, I randomly checked few files, the contents do have a key by the name `sentences` (some files have, some don't), but I couldn't find synopsis anywhere (or in any file I manually checked).

Is it because very few of the movies have synopsis?
If so, then is the key `synopsis` for it?
 
---------


[Update]
Nevermind, found it in the `meta` files.
